### ISSUE TITLE
Hf_Reformer got different accuracy results 2 eager runs

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#1262]
### 🐛 Describe the bug

```
python benchmarks/dynamo/torchbench.py --accuracy --float32 -d xpu -n10 --training  --only hf_Reformer --backend=inductor

xpu  train hf_Reformer                        
E0103 19:49:22.251000 2958475 site-packages/torch/_dynamo/utils.py:2369] Accuracy failed: allclose not within tol=0
E0103 19:49:22.252000 2958475 site-packages/torch/_dynamo/utils.py:2223] Accuracy failed for key name logits
eager_two_runs_differ

```

### Versions

Envirnoments:
Device: PVC 1100
torch-xpu-ops: 18bcd9a
python: 3.10
TRITON_COMMIT_ID: e98b6fcb8df5b44eb0d0addb6767c573d37ba024
TORCH_COMMIT_ID: b9fbd65dfd5e703bacbc6c25258d1215108b4faf
TORCHBENCH_COMMIT_ID: 766a5e3a189384659fd35a68c3b17b88c761aaac
TORCHVISION_COMMIT_ID: d23a6e1664d20707c11781299611436e1f0c104f
TORCHAUDIO_COMMIT_ID: b6d4675c7aedc53ba04f3f55786aac1de32be6b4
DRIVER_VERSION: 1.23.10.49.231129.50 (803.61)
KERNEL_VERSION: 5.15.0-73-generic #80-Ubuntu SMP Mon May 15 15:18:26 UTC 2023
BUNDLE_VERSION: 2025.0.1.20241113 (DL-Essential 2025.0.1)
OS_PRETTY_NAME: Ubuntu 22.04.2 LTS
GCC_VERSION: 11

**Comment by mengfei25** [comment]
Passed with latest code base
pytorch: 86be5d4421ebe96f147ecb145d4f416da727c958
torch-xpu-ops: ac1466c9893ea6a0c36daed711318accdb060e86
